scripts/benchmark_all.sh

```diff
@@ -74,18 +74,18 @@ do
   for benchmark in $benchmarks
   do
     echo "Running $benchmark for $commit... (single-threaded)"
-    ( ${build_folder}/$benchmark -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.log"
+    ( ${build_folder}/$benchmark --dont_cache_binary_tables -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.log"
```

Comment:
Should we take this PR as a reason to up the default SF to 10?

Reply:
👍 from my side. Defaulting to `SF 10` and add a second `SF 10 MT` run with the Pruning clustering?

Reply:
Sounds good

Reply:
Just to be clear: we also set the default of TPCH to SF 10 or only the benchmark_all script?

Reply:
Let's discuss this tomorrow, but I'm all in favor for setting it to SF 10 in hyriseBenchmarkTPCH.